### DISCUSSION TITLE
Admonitions

### DISCUSSION THREAD

**Discussion by rcdailey** [adityatelange/hugo-PaperMod#510] [Q&A] [answered]
Does papermod (or hugo) have anything equivalent to Mkdocs Admonitions? I've seen some hand-crafted versions using tables, but those look tedious. I'd be happy with a shortcode that provided the functionality. I did some googling but I can't find any out of the box functionality for this.

**Comment by pixelfabs** [comment]
Neither Hugo nor Papermod have one. **It looks very similar to how you would style `<blockquote>`.** I remember Grav CMS has it, but I had to look up what Admonitions are! I suggest to search for blockquote CSS. You can even get the CSS from Mkdocs and build a Hugo short code.
[http://boundedinfinity.github.io/2017/09/admonitions-in-css/](http://boundedinfinity.github.io/2017/09/admonitions-in-css/)

**Reply by smailzhu** [reply]
The admonitions mentioned by [Hugo contribute/documentation](https://gohugo.io/contribute/documentation/#admonitions) page.
Does it mean it will be a feature in hugo?

**Reply by danielfdickinson** [reply]
Shortcodes are no longer being added to Hugo binary, and there are discussions/efforts to move internal templates and existing shortcodes out of core (binary). You can however use the shortcodes you mentioned by grabbing them from the Hugo documentation site source code.

**Reply by ilovelinux** [reply]
New link: https://gohugo.io/contribute/documentation/#callouts

**Comment by pixelfabs** [comment, marked as answer]
Perhaps this is a better solution:
https://github.com/martignoni/hugo-notice

**Reply by moritz1000** [reply]
I know this is an older discussuion, but for anyone finding it in the future:
use https://github.com/KKKZOZ/hugo-admonitions instead.
Compared to hugo-notice, it respects theme switchting with PaperMod out of the box!
